src/Dfc.CourseDirectory.Core/DataStore/CosmosDb/Queries/GetApprenticeships.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq.Expressions;
+using Dfc.CourseDirectory.Core.DataStore.CosmosDb.Models;
+
+namespace Dfc.CourseDirectory.Core.DataStore.CosmosDb.Queries
+{
+    public class GetApprenticeships : ICosmosDbQuery<IDictionary<Guid, Apprenticeship>>
+    {
+        public GetApprenticeships(Expression<Func<Apprenticeship, bool>> predicate = null)
```

Comment:
We don't typically have constructors on DbQuery classes

Reply:
Any particular reason? It seems like the nicest place to put this.

Reply:
Consistency. Not super bothered; I foresee these types all becoming records soon :-)